app/models/match_decisions/eleven/eleven_hsa_acknowledges_receipt.rb

```diff
@@ -0,0 +1,153 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+module MatchDecisions::Eleven
+  class ElevenHsaAcknowledgesReceipt < ::MatchDecisions::Base
+    def to_partial_path
+      'match_decisions/eleven_hsa_acknowledges_receipt'
+    end
+
+    validate :cant_accept_if_match_closed
+    validate :cant_accept_if_client_has_related_active_match
+    validate :cant_accept_if_opportunity_has_related_active_match
+    validate :ensure_required_contacts_present_on_accept
+    validate :validate_cancel_reason
+
+    def label_for_status status
+      case status.to_sym
+      when :pending, :expiration_update then "New Match Awaiting Acknowledgment by #{Translation.translate('HSA Eleven')}"
+      when :acknowledged then "Match acknowledged by #{Translation.translate('HSA Eleven')}.  In review"
+      when :canceled then canceled_status_label
+      end
+    end
+
+    def started?
+      status&.to_sym == :acknowledged
+    end
+
+    def step_name
+      "New Match for #{Translation.translate('HSA Eleven')}"
+    end
+
+    def actor_type
+      Translation.translate('HSA Eleven')
+    end
+
+    def contact_actor_type
+      nil
+    end
+
+    def statuses
+      {
+        pending: 'Pending',
+        expiration_update: 'Pending',
+        acknowledged: 'Acknowledged',
+        declined: 'Declined',
+        canceled: 'Canceled',
+        destroy: 'Destroy',
+      }
+    end
+
+    def expires?
+      true
+    end
+
+    def editable?
+      super && saved_status !~ /acknowledged/
+    end
+
+    def initialize_decision! send_notifications: true
+      super(send_notifications: send_notifications)
+      update status: :pending
+      send_notifications_for_step if send_notifications
+    end
+
+    def accessible_by? contact
+      contact.user_can_act_on_behalf_of_match_contacts? ||
+      contact.in?(match.housing_subsidy_admin_contacts)
```

Comment:
We seem to have cleaned up `contact_actor_type` in some decisions.  Can you take a look at how we're using it in `app/models/match_decisions/approve_match_housing_subsidy_admin.rb`?  I think we want this to be:
```
contact.in?(match.send(contact_actor_type))
```
and 
```
def contact_actor_type
  nil
end
```
to be:
```
def contact_actor_type
  :housing_subsidy_admin_contacts
end
```

Looks like most (all?) of these decisions could be cleaned up that way.